### PR TITLE
streamkiste.tv

### LINKED ISSUE #1400: StreamKiste & KinoZ | StreamKiste.tv\KinoZ.to
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** konstey#0217 

**What's the name of the service?** kinoz.to and streamkiste.tv its the same like bs.to but you can watch movies 

**What should the Presence display?** watching a movie same like on bs.to or s.to 

**If possible, please provide a logo for the service (512x512 minimum)**

### PR DESCRIPTION
**Pull request by Venipa**
# StreamKiste.tv (german movie/series streaming portal)

### Searching
image

### Viewing, Playing & Pause - Movie
image
image
image

### Same for Series
image

### Default
image

closes #1400

### CODE REVIEW

**Comment by Venipa** [comment]
#1400

File: websites/S/StreamKiste/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "streamkiste.site",
+    "streamkiste.space",
+    "streamkiste.stream",
+    "streamkiste.party",
+    "streamkiste.press",
+    "streamkiste.website",
+    "streamkiste.download",
+    "streamkiste.network",
+    "streamkiste.online"
+  ],
```
Comment (Bas950) [inline websites/S/StreamKiste/dist/metadata.json:32]: maybe better to use regex?

File: websites/S/StreamKiste/presence.ts [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+            searching: "general.search",
+            searchFor: "general.searchFor"
+          },
+          newLang
+        );
+      }
+      const presenceData: PresenceData = {
+          largeImageKey: IMAGES.LOGO,
+          largeImageText: app.metadata.service
+        } as PresenceData,
```
Comment (Bas950) [inline websites/S/StreamKiste/presence.ts:177]: : PresenceData 
as PresenceData

Shouldnt be needed double??

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/S/StreamKiste/presence.ts (hunk 2 above)
Comment (Venipa) [reply websites/S/StreamKiste/presence.ts:177]: image
some properties do not exist, as PresenceData overrides the error and marks the obj valid

Comment (Bas950) [reply websites/S/StreamKiste/presence.ts:177]: Cause you cannot set the largeImageKey

**Comment by Venipa** [comment]
@Bas950 removed as Presence Data double, also using regexp for this now

**Comment by Bas950** [comment]
> @Bas950 removed as Presence Data double, also using regexp for this now

I saw you have other presences too with largeImageText, could you remove them please.

**Comment by Venipa** [comment]
@Bas950 removed them from my other branches:
#3377 
#3376 
#3375

**Review by ririxidev — APPROVED** [review]
lgtm

**Review by Bas950 — APPROVED** [review]
LGTM

**Pull request merged**